templates/scenarios/ts/spfx-tab/teamsapp.local.yml.tpl

```diff
@@ -26,6 +26,13 @@ configureApp:
     # Output: following environment variable will be persisted in current environment's .env file.
     # TEAMS_APP_ID: the id of Teams app
 
+  - uses: m365Title/acquire # Upload your app to Outlook and the Microsoft 365 app
```

Comment:
No need to add this action to `teamsapp.yml`?

Reply:
Good catch... It seems SPFx has no remote launch config as well.
Anyway, added to teamsapp.yml as well in case of further change.